src/Engines/Extensions/engine.multiRender.ts

```diff
@@ -198,12 +205,13 @@ ThinEngine.prototype.createMultipleRenderTarget = function (size: RenderTargetTe
     var textures: InternalTexture[] = [];
     var attachments: number[] = [];
 
-    var depthStencilBuffer = this._setupFramebufferDepthAttachments(generateStencilBuffer, generateDepthBuffer, width, height);
+    var useStencilTexture = this.webGLVersion > 1 && generateDepthTexture && options.depthTextureFormat == Constants.TEXTUREFORMAT_DEPTH24_STENCIL8;
+    var depthStencilBuffer = this._setupFramebufferDepthAttachments(!useStencilTexture && generateStencilBuffer, !generateDepthTexture && generateDepthBuffer, width, height);
 
     rtWrapper._framebuffer = framebuffer;
     rtWrapper._depthStencilBuffer = depthStencilBuffer;
-    rtWrapper._generateDepthBuffer = generateDepthBuffer;
-    rtWrapper._generateStencilBuffer = generateStencilBuffer;
+    rtWrapper._generateDepthBuffer = !generateDepthTexture && generateDepthBuffer;

```

Comment:
If I'm interpreting the spec correctly, there's no point in generating a stencil buffer here either:

    FRAMEBUFFER_UNSUPPORTED if depth and stencil attachments, if present, are not the same image. 

3.7.4 https://www.khronos.org/registry/webgl/specs/2.0/

Reply:
i.e., both _generateDepthBuffer and _generateStencilBuffer should be gated on the same condition (!_generateDepthTexture)

Reply:
@sebavan if you have any thoughts on this let me know. I'm actually going to leave it how it is in the PR. I realized if I disable the stencil render buffer when depth texture is enabled, that's a breaking change as previous code allowed for this too. And FWIW, I haven't been able to repro any GL errors by binding a stencil render buffer and a depth(/stencil) texture at the same time, so it's unclear to me if that is actually against the spec or not.